lichess-bot.py

```diff
@@ -180,7 +180,8 @@ def log_proc_count(change, queued, used):
                 logger.warning("Unable to handle response from lichess.org:")
                 logger.warning(event)
                 if event.get("error") == "Missing scope":
-                    logger.warning('Please check that the API access token for your bot has the scope "Play games with the bot API".')
+                    logger.warning('Please check that the API access token for your bot has the scope "Play games '
+                                   'with the bot API".')
```

Comment:
If you want to split the string to keep the lines under the length limit, something like this would work:
```python
                    logger.warning('Please check that the API access token for'
                                   ' your bot has the scope "Play games with'
                                   ' the bot API".')
```

Reply:
This change is enough to make it under the length limit. There is no reason to break it even more.